File: archive/Java/프로그래머스/0/120831. 짝수의 합/짝수의 합.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int sum = 0;
        for(int i = 0; i<=n; i++) {
            sum = (i % 2 ==0 ) ? i : 0;
            answer += sum;
        }
        
        return answer;
    }
}

File: C/프로그래머스/0/120831. 짝수의 합/짝수의 합.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int sum=0;
    int i;
    for(i=1; i<=n; i++) 
    {
        if(i%2==0) {
        printf("%d",i);
        sum += i;
        }
        
    }
    
    return sum;
}